spec/controllers/v0/disability_compensation_forms_controller_spec.rb

```diff
@@ -39,6 +40,7 @@
     context 'lighthouse' do
       before do
         Flipper.enable('disability_compensation_lighthouse_brd')
+        Flipper.disable('disability_compensation_staging_lighthouse_brd')
```

Comment:
should we add a test at this level to make sure the controller sends back and calls the right provider at this stage? 🤔